payments/utils.py

```diff
@@ -175,14 +176,21 @@ def check_payments_status(verbose=0):
             _process_payment(order, payment, verbose)
 
 
+def get_unpaid_orders(overdue=False):
+    deadline = timezone.now() - timedelta(days=UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY)
+    orders = Order.objects.filter(Q(status=Order.AWAITING) | Q(status=Order.PARTLY_PAID))
+    orders = orders.filter(Q(unpaid_notification_sent_amount=0) & Q(date_created__lt=deadline) |
```

Comment:
I am not sure whether this means (A and B) or C  ...or... A and (B or C). Could you put there the parenthesis explicitly?